build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 subprojects {
 
     group = 'edu.wisc.my.restproxy'
-    version = '3.1.0'
+    version = '3.0.0'
```

Comment:
3.0.0 already went in https://github.com/UW-Madison-DoIT/rest-proxy/commit/89ed5e2647c593032a48e77c15f4a8ceb2d1e294

Looking at maven central: http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22edu.wisc.my.restproxy%22%20AND%20a%3A%22rest-proxy-core%22